### PR TITLE
feat(forms): add `Value.ArraySelection` component

### PR DESCRIPTION
**Pull request by joakbjerk**

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Nice to have this new value component 🙏

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<Examples.LabelAndValue />
+
+### Inline
+
+<Examples.Inline />
+
+{/* This is not yet supported */}
+{/* ### Field.ArraySelection with path */}
+{/* When using the same `path` as on a `Field.ArraySelection`, the title will be used as the dispalyed value. */}
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx:35]:
```suggestion
{/* When using the same `path` as on a `Field.ArraySelection`, the title will be used as the displayed value. */}
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<Examples.Inline />
+
+{/* This is not yet supported */}
+{/* ### Field.ArraySelection with path */}
+{/* When using the same `path` as on a `Field.ArraySelection`, the title will be used as the displayed value. */}
+{/* <Examples.FieldArraySelectionPath /> */}
+
+### Field.Option and Field.ArraySelection
+
+When using the same `path` as on a `Field.ArraySelection`, the `Field.Option` title will be used as the dispalyed value.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx:40]:
```suggestion
When using the same `path` as on a `Field.ArraySelection`, the `Field.Option` title will be used as the displayed value.
```

**Review by langz — APPROVED** [review]

**Pull request merged**